src/hypothesis/control.py

```diff
@@ -81,7 +81,7 @@ def close(self):
         for task in self.tasks:
             try:
                 task()
-            except:
+            except BaseException:
```

Comment:
I actually think this one might want to be an Exception rather than a BaseException, but I'm OK with leaving the behaviour unchanged for now while we think it through.

Reply:
Yep, my approach was "BaseException unless I'm certain it's not"